packages/expo-cli/e2e/__tests__/install-test.ts

```diff
@@ -0,0 +1,98 @@
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs-extra';
+import path from 'path';
+import temporary from 'tempy';
+
+import { EXPO_CLI } from '../TestUtils';
+
+const projectRoot = temporary.directory();
+
+function getRoot(...args) {
+  return path.join(projectRoot, ...args);
+}
+
+// 3 minutes -- React Native takes a while to install
+const extendedTimeout = 3 * 1000 * 60;
+
+beforeAll(async () => {
+  jest.setTimeout(extendedTimeout);
+  await fs.mkdirp(projectRoot);
+});
+
+afterAll(async () => {
+  await fs.remove(projectRoot);
+});
```

Comment:
```suggestion
```
No need to clean up since it's in a temp directory. The OS will clean it up. This is also nicer in cases where you need to debug the test and look at the project that was generated.